src/applications/appeals/10182/tests/utils/replace.unit.spec.js

```diff
@@ -60,4 +64,11 @@ describe('fixDateFormat', () => {
     expect(fixDateFormat('2023-10-1')).to.eq('2023-10-01');
     expect(fixDateFormat('2000-6-30')).to.eq('2000-06-30');
   });
+  it('should return properly formatted date string when passed dates with weird spacing', () => {
+    expect(fixDateFormat('2020--')).to.eq('2020-00-00');
+    expect(fixDateFormat('2020-1-')).to.eq('2020-01-00');
```

Comment:
same question here. Would a `00` for the day still return a valid date?

Reply:
It's not a valid date. We want it to return `00` so the validation error kicks in - if we forced this to be `01`, the resulting  submitted date may not match what the Veteran is trying to enter.